config/environment.py

```diff
@@ -44,29 +47,32 @@
     UPLOADS_DEFAULT_DEST = os.environ.get('STAGING_UPLOADS_DEFAULT_DEST')
     UPLOADS_DEFAULT_URL = os.environ.get('STAGING_UPLOADS_DEFAULT_URL')
 
-# see secrets.py.example for the rest of the config values that need
-# to be modified for localhost development
 if LOCALHOST:
+    GITHUB_CLIENT_ID = os.environ.get('GITHUB_CLIENT_ID') or os.environ.get('FAKE_ID')  # noqa
+    GITHUB_CLIENT_SECRET = os.environ.get('GITHUB_CLIENT_SECRET') or os.environ.get('FAKE_SECRET')  # noqa
```

Comment:
These two lines will need to be changed in the future. So we have a proper testing configuration.

Basically 
* ConfigProduction (for production server)
* ConfigStaging (for hosting on staging)
* ConfigDevelopment (for localhost dev)
* ConfigTesting (for circleCI)

Reply:
To be clearer localhost will be scrapped in the future. 
And we can possibly add a kind of docker config.